integration/runner/runner.go

```diff
@@ -141,9 +171,20 @@ func PushAndRunTests(host, testDir string) error {
 		}
 	}
 	if err != nil {
-		err = fmt.Errorf("error on host %s: %v", host, err)
+		err = fmt.Errorf("error on host %s: %v\n%+v", host, err, attributes)
+		// Copy logs from the host
+		args = common.GetGCComputeArgs("copy-files", fmt.Sprintf("%s:%s/log.txt", host, testDir), "./")
+		err = RunCommand("gcloud", args...)
+		if err != nil {
+			errChan <- fmt.Errorf("error fetching logs: %v", err)
```

Comment:
Is there another routine listening to this channel?  If not this will block forever.